src/front/app/components/Challenges/index.js

```diff
@@ -3,10 +3,34 @@ import ChallengeList from './ChallengeList';
 import Menu from './Menu';
 import Navigation from './../Navigation';
 
-export default () => (
-    <div>
-        <Menu />
-        <ChallengeList />
-        <Navigation />
-    </div>
-);
+class Challenges extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            menuClass: 'challenges__menu'
```

Comment:
Classes should not be stored in the state (again, for separation of concerns, classes and other dom-related attributes should be handled and encapsulated only in render function), you can store indicators based on which render function may decide which class to apply